Logging/private/Start-LoggingManager.ps1

```diff
@@ -0,0 +1,67 @@
+#Requires -Version 5.0
```

Comment:
@tosoikea @EsOsO This would be a breaking change for people that are using Logging in PS 4 and lower.

Is this desired at this time? Also this goes in contradiction the `#requires -Version 4` in Logging.psm1 file

Reply:
I am quickly going to remove it. I just used it while writing code for using namespaces.